### PR TITLE
LightSpotTextureProjectionRenderer;

### PR DESCRIPTION
**Pull request by nehrk**
LightSpotTextureProjectionRenderer todo's implementation.

### CODE REVIEW

File: sources/engine/Xenko.Engine/Rendering/Lights/ITextureProjectionRenderer.cs [hunk 1]
```diff
@@ -0,0 +1,13 @@
+// Copyright (c) Xenko contributors (https://xenko.com)
+// Distributed under the MIT license. See the LICENSE.md file in the project root for more information.
+
+namespace Xenko.Rendering.Lights
+{
+    /// <summary>
+    /// Interface to project a texture onto geometry.
+    /// </summary>
+    internal interface ITextureProjectionRenderer // TODO: Move to separate file! // UPD 04.08.18 - moved.
```
Comment (xen2) [inline sources/engine/Xenko.Engine/Rendering/Lights/ITextureProjectionRenderer.cs]: Please remove TODO/UPD comment completely, git has history if we need it.

File: sources/engine/Xenko.Engine/Rendering/Lights/ITextureProjectionShaderGroupData.cs [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+// Distributed under the MIT license. See the LICENSE.md file in the project root for more information.
+
+using Xenko.Core.Collections;
+using Xenko.Core.Mathematics;
+using Xenko.Engine;
+using Xenko.Shaders;
+
+namespace Xenko.Rendering.Lights
+{
+    public interface ITextureProjectionShaderGroupData // TODO: Move to separate file! // UPD 04.08.18 - moved.
```
Comment (xen2) [inline sources/engine/Xenko.Engine/Rendering/Lights/ITextureProjectionShaderGroupData.cs]: Please remove TODO/UPD comment completely, git has history if we need it.

**Comment by nehrk** [comment]
Hi @Kryptos-FR, thank you for useful link.

File: sources/engine/Xenko.Engine/Rendering/Lights/ITextureProjectionShaderGroupData.cs (hunk 2 above)
Comment (nehrk) [reply sources/engine/Xenko.Engine/Rendering/Lights/ITextureProjectionShaderGroupData.cs]: Done.

**Pull request closed without merging**